ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
You can use something called object to generate dummy objects. I created a gist for you. I didn't test it though. https://gist.github.com/ravisantoshgudimetla/8fe92f88f433ef2ceaee7100c5ea6b70. I don't know the further implications of it from security stand point but thats a good starting point is what I think.